src/main/java/com/gooddata/executeafm/result/ExecutionResult.java

```diff
@@ -61,17 +61,17 @@ public ExecutionResult(final String[][] data, final Paging paging) {
      * Creates new result
      * @param data result data
      * @param paging result paging
-     * @param attributeHeaderItems items for attribute headers, for each attribute in each dimension, there is a list of headers
+     * @param headerItems items for attribute headers, for each attribute in each dimension, there is a list of headers
```

Comment:
Please fix the doc - not only attribute headers can be in the 3D list now.

Reply:
thx for catch